Modules/CalcOffence-3_0.lua

```diff
@@ -2020,8 +2036,9 @@ function calcs.offence(env, actor, activeSkill)
 			if baseVal > 0 then
 				skillFlags.shock = true
 				output.ShockDurationMod = 1 + skillModList:Sum("INC", cfg, "EnemyShockDuration") / 100 + enemyDB:Sum("INC", nil, "SelfShockDuration") / 100
+				output.ShockEffectMod = skillModList:Sum("INC", cfg, "EnemyShockEffect")
 				if breakdown then
-					t_insert(breakdown.ShockDPS, s_format("For shock to apply, target must have no more than %d life.", baseVal * 20 * output.ShockDurationMod))
+					t_insert(breakdown.ShockDPS, s_format("For shock to apply, target must have no more than %d life.", baseVal * 250 * output.ShockDurationMod * (1 + output.ShockEffectMod / 100)))
```

Comment:
This formula is incorrect:
```
.05 = (1+increased_effect/100) * 1/2(damage_dealt/ailment_threshold)^(.4)
```
which implies
```
ailment_threshold = -((100 + increased_effect)^(5/2) damage_dealt)/(100 sqrt(10))
```

Also please update the wording to indicated 
 - minimum shock of 5%, 
 - target must have no more than %d ailment threshold